pkg/query-service/constants/constants.go

```diff
@@ -90,7 +90,7 @@ func EnableHostsInfraMonitoring() bool {
 	return GetOrDefaultEnv("ENABLE_INFRA_METRICS", "true") == "true"
 }
 
-var KafkaSpanEval = GetOrDefaultEnv("KAFKA_SPAN_EVAL", "false")
+var KafkaSpanEval = GetOrDefaultEnv("KAFKA_SPAN_EVAL", "true")
```

Comment:
intended change ?

Reply:
yes

Reply:
yes

Reply:
I am re-iterating my position on this as mentioned here https://github.com/SigNoz/signoz/pull/6661#issuecomment-2550840426, this change should not be enabled by default. Please revert it.